scanner/Makefile

```diff
@@ -190,6 +179,7 @@ image-scanner: $(if $(CI),,image/scanner/bin/scanner) \
 	@echo "+ $@"
 	$(DOCKERBUILD) \
         -t stackrox/$(image-prefix):$(TAG) \
```

Comment:
I think we can probably just replace this with the new line you added. What do you think @jvdm ?